src/config/sierraleone/layers.json

```diff
@@ -17,6 +17,371 @@
       }
     }
   },
+  "admin1_boundaries": {
+    "type": "boundary",
+    "path": "data/sierraleone/sle_admin1new_boundary_20200309.json",
+    "opacity": 0.8,
+    "admin_code": "adm1_id",
+    "admin_level_names": ["adm1_name"],
+    "admin_level_local_names": ["adm1_name"],
+    "styles:": {
+      "fill": {
+        "fill-opacity": 0
+      },
+      "line": {
+        "line-color": "gray",
+        "line-width": 3,
+        "line-opacity": 0.8
+      }
+    }
+  },
+  "rainfall_dekad": {
+    "title": "10-day rainfall estimate (mm)",
+    "type": "wms",
+    "server_layer_name": "rfh_dekad",
+    "additional_query_params": {
+      "styles": "rfh_16_0_300"
+    },
+    "base_url": "https://api.earthobservation.vam.wfp.org/ows/",
+    "date_interval": "days",
+    "opacity": 0.7,
+    "legend_text": "Estimate of precipitation over a 10-day period derived from CHIRPS (UCSB Climate Hazards Group). https://www.chc.ucsb.edu/data/chirps",
+    "legend": [
+      { "value": 0, "label": "0 mm", "color": "#fafafa" },
```

Comment:
Would it be easy to limit the number of labels in these legends?
Maybe merge a few that are out of usual range for this country?

Reply:
We can't limit this color ramp any further but will limit where we can which is mostly on the temperature layers